tools/install.sh

```diff
@@ -48,6 +48,24 @@ case $os in
       echo
       echo "Please visit https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard for Distrowise/Distro-specific Install Instructions."
       echo
-      ;;
+      
+      echo "Do you want to see the instructions for installing OpenBangla Keyboard in your system?"
+      select yn in "Yes" "No"; do
+      
+      case $yn in
+            
+            (Yes)
+                  URL="https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard"
+                  [[ -x $BROWSER ]] && exec "$BROWSER" "$URL"
+                  path=$(which xdg-open || which gnome-open) && exec "$path" "$URL"
```

Comment:
ShellCheck shows warning for using `which` https://github.com/koalaman/shellcheck/wiki/SC2230 
Yoy can check it yourself https://www.shellcheck.net/

Reply:
`which` is bad because it is not POSIX compatible but it does not matter much because all recent distro has it. But, I'll change it for best practice.